### ISSUE TITLE
Make docstrings uniform

### ISSUE THREAD

**Issue by mee5-pdx** [oresat/oresat-adcs-software#14]
Many functions only take a few lines, so formatting docstrings like in the google docstring format may be nice

**Comment by dankeg** [comment]
I'm interested in taking a crack at this, to clarify this would involve transitioning to this format?: [https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e)

**Comment by mee5-pdx** [comment]
> I'm interested in taking a crack at this, to clarify this would involve transitioning to this format across the repo?: https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e

Yes, I was aiming to do a similar format to this. The docstrings from lines 67-75 is what I would prefer most docstrings to look like

- A short summary/description
- An args/parameters section (with types specified)
- A returns section (with types specified)

See https://realpython.com/documenting-python-code/#docstring-formats
I generally disliked the numpy/scipy type docstrings because it basically doubles the number of lines.
